client/src/routes/fight/bar/FightTime.svelte

```diff
@@ -59,48 +59,53 @@
 
 <div>
     <button on:click={() => dispatch('return')}>
-        <Icon icon="fe:arrow-left" height="3em" />
+        <Icon icon="bx:left-arrow-circle" color="#2f4858" height="6vh"/>
     </button>
 
-    <p> {showTime(current)} </p>
+    <div class="info">
+        {showTime(current)}
+    </div>
 
     <button class="previous" on:click={paused ? startTimer : pauseTimer}>
         {#if paused}
-            <Icon icon="fe:play" height="3em" />
+            <Icon icon="bx:play-circle" color="#2f4858" height="6vh"/>
         {:else}
-            <Icon icon="fe:pause" height="3em" />
+            <Icon icon="carbon:pause-outline" color="#2f4858" height="6vh"/>
         {/if}
     </button>
 </div>
 
 <style>
     div {
         display: flex;
+        height: 7.5vh;
```

Comment:
To jest wspomniane wcześniej wystąpienie ustawienia wysokości. 
Przy okazji, uszczegółowiłbym selektor, po wprowadzeniu nowego (wątpliwie potrzebnego), `div`a.